public/sass/_grafana.scss

```diff
@@ -12,7 +12,6 @@
 @import 'utils/widths';
 
 // COMPONENTS
-@import 'components/buttons';
```

Comment:
I see these styles are used in some old html files, but also in `diff-view.ts`, so not sure it's safe to remove them.

Reply:
I suck at searching files 😅 You're right.